packages/charts/src/charts/table/row.component.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Ceil, Typography } from '@keen.io/ui-core';
+import { StripedTable } from './table.styles';
+import { copyToClipboard } from './table.utils';
+
+type Props = {
+  data?: any;
```

Comment:
The same here. Based on code - the data is an `Record` with `string` keys and string | number value.